Documentation/Packaging/MRTK_Packages.md

```diff
@@ -20,43 +52,66 @@ The Mixed Reality Toolkit Foundation is the set of code that enables your applic
 
-* **Platform Providers**
+**Platform providers**
 
-The MRTK Platform Provider packages are the components that enable the Mixed Reality Toolkit to target Mixed Reality hardware and platform functionality.
+The platform providers are the components that enable the Mixed Reality Toolkit to target Mixed Reality hardware and platform functionality.
 
 Supported platforms include:
 
 - Windows Mixed Reality
+- Oculus
 - OpenVR
-- Windows Voice
+- Mobile AR
+- Leap Motion
 
-* **System Services**
+**Services**
 
-Core services provide the default implementations for the system service interfaces, defined in the core package.
+The services folder contains the core systems that provide the default implementations for the system service interfaces.
 
 The MRTK foundation includes the following system services:
 
 - [Boundary System](../Boundary/BoundarySystemGettingStarted.md)
+- [Camera System](../CameraSystem/CameraSystemOverview.md)
 - [Diagnostic System](../Diagnostics/DiagnosticsSystemGettingStarted.md)
 - [Input System](../Input/Overview.md)
 - [Spatial Awareness System](../SpatialAwareness/SpatialAwarenessGettingStarted.md)
 - [Teleport System](../TeleportSystem/Overview.md)
 
-* **Feature Assets**
+**Standard Assets**
+
+The MRTK standard assets is a collection of components that are recommended for all mixed reality experiences, including:
 
-Feature Assets are collections of related functionality delivered as Unity assets and scripts including user interface controls, Standard assets, and more.
+- MRTK Standard shader
+- Basic materials using the MRTK Standard shader 
 
-## Extensions package
+### Extensions package
 
-The extensions package contains additional services and components that extend the functionality of the foundation package.
+The extensions package contains additional services and components that extend the functionality of the MRTK.
 
+- [Hand Physics Service](../Extensions/HandPhysicsService/HandPhysicsServiceOverview.md)
+- Lost Tracking Service
 - [Scene Transition Service](../Extensions/SceneTransitionService/SceneTransitionServiceOverview.md)
 
-## Examples package
+### Tools package
+
+The tools package contains tools that are useful for creating mixed reality experiences. In general, these tools are editor components and their code does not ship as part of an application.
+
+Some of the tools in this package include:
+
+- [Dependency Window](../Tools/DependencyWindow.md)
+- [Extension Service Creation Wizard](../Tools/ExtensionServiceCreationWizard.md)
+- [Optimize Window](../Tools/OptimizeWindow.md)
+- [Screenshot Utility](../Tools/ScreenshotUtility.md)
+
+### Test utlities package
+
+The test utilities package contains a collection of helper scripts that enable developers to easily create play mode tests. These utilities are especially useful for developers creating MRTK components.
```

Comment:
Nit: Consider adding a link to:

https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/UnitTests.html#play-mode-tests

For examples on the types of test you could write with these utilities.